README.md

```diff
@@ -49,7 +49,7 @@ dotnet ef database update --context OrganisationRegistryContext
 #### To copy events from staging to local db:
 
 ```bash
-bcp OrganisationRegistry.OrganisationRegistry.Events out ./events.bcp -S<server> -U<user> -n -E
+bcp OrganisationRegistry.Events out ./events-staging.bcp -S<staging-ip>,<staging-port> -U <username> -d organisation-registry -n -E
 bcp OrganisationRegistry.OrganisationRegistry.Events in ./events.bcp -S 127.0.0.1,21433 -U sa -P E@syP@ssw0rd -n -E
```

Comment:
this should change to `events-staging.bcp` as well, right?

Reply:
klopt